src/App.jsx

```diff
@@ -4,9 +4,19 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  filmQueryChange= (event) => {
+    this.setState({ query: event.target.value });
+  };
 
   render() {
+    const { query } = this.state;
+    const normalizedQuery = query.toLowerCase();
+    const visibleFilm = moviesFromServer.filter(film => film.title.toLowerCase().includes(normalizedQuery));
```

Comment:
Read more carefully task description:

> 2. Filter the movies using state.query by searching it in the movie.title or movie.description

You should filter not only by title but also by description